torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -1602,12 +1605,21 @@ def call_function(
             scale,
         ) = self.normalize_to_args(args, kwargs)
 
-        p_args = self.create_wrapped_node(tx, query, score_mod)
+        score_mod_node, score_mod_lifted_args = self.create_wrapped_node(
+            tx, query, score_mod, "score_mod"
+        )
+        mask_fn = block_mask.items[-1]
+        if isinstance(mask_fn, ConstantVariable):
```

Comment:
What does this do?

Reply:
This is going to handle the case when mask_fn is None in eager mode.